0015-automatic-cluster-upgrades/0015-automatic-cluster-upgrades.md

```diff
@@ -0,0 +1,61 @@
+# RFC 0015 - Automatic Cluster Upgrades
+
+As a user, and also as KaaS team provider, I would like to have the clusters running in the latest version automatically, and not worry about CVEs or EOL of the cluster components.
+
+As a user I would like to define maintenance windows to ensure cluster are not upgraded out of these hours.
```

Comment:
```suggestion
As a user, I would like to define maintenance windows to ensure clusters are not upgraded outside of these hours.
```

Reply:
In a way, you could think of either giving maintenance windows or do it the other way around and say "these times are not good". I know enterprise is rather used to former, but we could evaluate if latter might not also be feasible, to go towards "it should always be ok, and only exceptions to that are marked". Might also be sth for a future iteration though.

Reply:
yeah we could come with a format the could include or exclude dates/times